source/app/utils/utils.ts

```diff
@@ -0,0 +1,4 @@
+export const formatCapitalLetters = (value: string) => {
```

Comment:
I think, this utility should have different name, because the transformation is only change the first letter capitalize and remove all dashes or make a other thing.

Keep in main this, one function called "format" and to have multiple options, somethi like

format("text-to-transfor", formater.FirstLetterCapitalizeAndSpaces);
so, formater (or whatever enum name) is a enum with different options and method inside a switch for different cases.